docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,311 @@
+will be how we move to a system that has the technology in place to
+verify connections? The answer to that is what this series calls
+"bootstrapping trust."
+
+### Ziti & Zero Trust
+
+In a Zero Trust model, there needs to exist mechanisms to verify
+identities such that trust can be granted. Zero Trust does not mean
+there is no trust. Zero Trust means that trust is given only after
+verification. Even then, that trust is limited to accessing the minimum
+network resources necessary. To accomplish this, we need a network that
+can force all connections through the following process.
+
+1. Authenticate
+2. Request Access To A Resource
+3. Connect To The Requested Resource
+
+To help make Zero Trust and Bootstrapping trust a bit clearer, it helps
+to have a concrete system to use an example. It just so happens that the
+Ziti software system makes a great example! Let us examine Ziti.
+
+In Ziti, all of the above steps require interacting with a Ziti
+Controller. The Ziti Controller manages the Ziti overlay network. It
+maintains a list of known network services, SDK clients, routers,
+enrollments, policies, and much more! All of these pieces working
+together to create a Ziti Network. A Ziti Network is an overlay network
+\- meaning it creates a virtual network on top of some other network
+referred to as the underlay network.
+
+In the Ziti Network, all network resources are modeled as services in
+the Ziti Controller. All services on a Ziti Network are only accessible
+via the Ziti Network. Network services can be made available via a Ziti
+Network in a variety of manners. The preferred method is embedding the
+Ziti SDK inside of applications and servers as it provides the highest
+degree of Zero Trust security. However, it is also possible to configure
+various overly-to-underlay connections to existing network services via
+"router termination" or a particular type of application with the Ziti
+SDK embedded in it called a "tunneler."
+
+The Ziti Controller also knows about one or more Ziti Routers that form
+a mesh network that can create dynamic circuits amongst themselves.
+Routers use circuits to move data across the Ziti Network. Routers can
+be configured to allow data to enter and exit the mesh. The most common
+entry/exit points are Ziti SDKs acting as clients or servers.
+
+Network clients wishing to attach the network use the Ziti SDK to first
+authenticate with the Ziti Controller. During authentication, the Ziti
+SDK client and Ziti Controller will verify each other. Upon successful
+authentication, the Ziti Controller can provide a list of available
+services to dial (connect) or to bind (host) for the authenticated SDK
+Client. The client can then request to dial or bind a service. If
+fulfilled, a session is associated with the client and service. This new
+session is propagated to the necessary Ziti Routers, and the required
+circuits are created. The requesting client is returned a list of Ziti
```

Comment:
> The requesting client is returned a list of Ziti Routers that can be connected to complete the last mile

this reads off. i'd restructure it. maybe: "The client is returned the list of Ziti Routers which can be connected to in order to complete the last mile" ?  Or maybe you need another 'to' in your original sentence but "The requesting client is returned a list of Ziti Routers that can be connected to to complete the last mile" reads off to me still

Reply:
Took yours.